components/AlternateHrefLangElements.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Rainlab\Translate\Components;
+
+use Cms\Classes\ComponentBase;
+use RainLab\Translate\Classes\Translator;
+use RainLab\Translate\Models\Locale as LocaleModel;
+use October\Rain\Router\Router as RainRouter;
+
+class AlternateHrefLangElements extends ComponentBase
+{
+
+
+    public function componentDetails()
+    {
+        return [
+            'name' => 'Alternate HREF Lang elements',
```

Comment:
Provide language keys for this component's name and description please, it would be funny if the Translation plugin didn't actually provide a translation for itself :)

Reply:
That would be strange indeed ;)
Adding them as we speak!